<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Realtime</class>
 <widget class="QWidget" name="Realtime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>1171</width>
     <height>811</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QGraphicsView" name="front_right"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string> Front center camera</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QGraphicsView" name="bottom_center"/>
    </item>
    <item row="1" column="0">
     <widget class="QGraphicsView" name="front_left"/>
    </item>
    <item row="5" column="1">
     <widget class="QGraphicsView" name="back_center"/>
    </item>
    <item row="1" column="1">
     <widget class="QGraphicsView" name="front_center"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string> Front right camera</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string> Front left camera</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string> Bottom center camera</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string> Back center camera</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
